helm/app-updater-app/templates/cronjob.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   labels:
     app:  {{ template "app-updater.fullname" . }}
 spec:
-  schedule: {{ .Values.schedule }}
+  schedule: {{ .Values.schedule | quote }}
```

Comment:
just curious, why is it necessary to quote the schedule if the provided values in `values.yaml` is already quoted? Helm is so weird sometimes.